Pocket.Logger/For.MicrosoftExtensionsLogging/LoggerFactoryExtensions.cs

```diff
@@ -48,7 +48,7 @@ private class Logger : ILogger
                     exception: exception,
                     category: category);
 
-                if (state is FormattedLogValues formattedLogValues)
+                if (state is IReadOnlyList<KeyValuePair<string, object>> formattedLogValues)
```

Comment:
`FormattedLogValues` is defined as `internal` so I replaced it with its base type:
```c#
namespace Microsoft.Extensions.Logging
{
    internal readonly struct FormattedLogValues : IReadOnlyList<KeyValuePair<string, object>>
    {
    }
}
```